src/server/services/govuk-notify.ts

```diff
@@ -131,17 +131,184 @@ export async function sendEditDetailsEmail(
 
     message = message.replace(/(?:\r\n)/g, "\n^");
 
-    await getNotifyClient().sendEmail(config.GOVUK_NOTIFY_EDIT_DETAILS_TEMPLATE_ID?.trim(), emailAddress, {
-      personalisation: {
-        typeSingular,
-        typePlural,
-        contactName,
-        message,
-        changeLink,
-      },
-    });
+    await getNotifyClient().sendEmail(
+      NOTIFY.templates.edit,
+      emailAddress, {
+        personalisation: {
+          typeSingular,
+          typePlural,
+          contactName,
+          message,
+          changeLink,
+        },
+      });
   } catch (error) {
-    logger.error(`Unable to send change request email`);
     throw new Error(`Unable to send change request email: ${error.message}`);
   }
 }
+
+export async function sendAnnualReviewPostEmail(
+  milestoneTillAnnualReviewStart: MilestoneTillAnnualReview,
+  emailAddress: string,
+  typePlural: string,
+  country: string,
+  annualReviewDate: string
+): Promise<{ result?: boolean; error?: Error }> {
+  if (config.isSmokeTest) {
+    logger.info(`isSmokeTest[${config.isSmokeTest}], would be emailing to ${emailAddress}`);
+    return { result: true };
+  }
+
+  const notifyTemplates: Record<MilestoneTillAnnualReview, string> = {
+    POST_ONE_MONTH: NOTIFY.templates.annualReviewNotices.postOneMonth ?? "",
```

Comment:
aren't all these values casted to empty string in config anyway?

Reply:
yep, good spot